openpype/client/mongo/entities.py

```diff
@@ -0,0 +1,1509 @@
+"""Unclear if these will have public functions like these.
+
+Goal is that most of functions here are called on (or with) an object
```

Comment:
I won't do any changes in mongo subfolder unless the code changed by this PR. It's because of easy merge to this branch. We can do this changes in develop...